src/kagglehub/config.py

```diff
@@ -114,3 +120,19 @@ def _get_kaggle_credentials_folder() -> str:
 
 def _is_env_var_truthy(env_var_name: str) -> bool:
     return env_var_name in os.environ and os.environ[env_var_name].lower() in TRUTHY_VALUES
+
+
+def set_kaggle_credentials(username: str, api_key: str):
+    error_message = "Both username and API key cannot be empty or whitespace"
+    stripped_username = username.strip()
+    stripped_api_key = api_key.strip()
+    if not stripped_username or not stripped_api_key:
+        raise ValueError(error_message)
```

Comment:
nit: You can define the error_message inside the if, just above the call to `raise`.